### PR TITLE
ISS-312 - STRICT compare payload

### PR DESCRIPTION
**Pull request by nirmalchandra**

### CODE REVIEW

File: core/src/test/java/org/jsmart/zerocode/core/domain/StepTest.java [hunk 1]
```diff
@@ -161,7 +168,126 @@ public void shouldSerializeSingleStep_method() throws Exception {
...
+                "            \"type\": \"home\",\n" +
+                "            \"line1\": \"10 Random St\"\n" +
+                "        }\n" +
+                "    ]\n" +
+                "}";
+        try {
+            JSONAssert.assertEquals(expected, actual, JSONCompareMode.STRICT);
+
+        } catch (Throwable ex) {
+            System.out.println("Caught: " + ex);
```
Comment (vkjha2000) [inline core/src/test/java/org/jsmart/zerocode/core/domain/StepTest.java]: Can you assert the message content?
> Remove sysout please

**Comment by authorjapps** [comment]
solves #312

File: core/src/test/java/org/jsmart/zerocode/core/domain/StepTest.java [hunk 2]
```diff
@@ -161,7 +168,126 @@ public void shouldSerializeSingleStep_method() throws Exception {
         JsonNode singleStepNode = mapper.valueToTree(stepJava);
         String singleStepNodeString = mapper.writeValueAsString(singleStepNode);
 
-        JSONAssert.assertEquals(jsonDocumentAsString,singleStepNodeString, false);
+        JSONAssert.assertEquals(jsonDocumentAsString, singleStepNodeString, false);
         assertThat(singleStepNode.get("method").asText(), is("POST"));
     }
+
+    @Test
+    public void testPayLoad_strictPass() {
```
Comment (officiallysameer) [inline core/src/test/java/org/jsmart/zerocode/core/domain/StepTest.java]: Add a unit test for STRICT array matching for example
if expected and actual array has only one element with same values, then only the matching will pass

Comment (nirmalchandra) [reply core/src/test/java/org/jsmart/zerocode/core/domain/StepTest.java]: done

**Pull request merged**